lib/document/enrichContractWithBaseDocument.js

```diff
@@ -26,7 +26,9 @@ function enrichContractWithBaseDocument(contract) {
 
```

Comment:
Clone and update (`properties`, `required`) base document according to `excludeBaseProprety `